webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
My only hesitation is that it seems to hinge on there being no more arguments. E.g. `getStats(undefined, onSuccess, onFailure);` just seems wrong for some reason.